NOnion/Utility/HiddenServicesUtility.fs

```diff
@@ -34,6 +34,22 @@ module HiddenServicesUtility =
 
         unixNow - timeElapsedSinceStartOfRun |> FromUnixTimestamp
 
+    let GetStartTimeOfPreviousSRVProtocolRun
+        (now: DateTime)
```

Comment:
why is this param called now? I think it needs a better name

Reply:
@aarani

Reply:
There's already a function just above this that has the same name, I prefer to be consistant.

Reply:
sure, be consistent in that commit, but then rename it in a next commit in both functions